nb/uihandler.exceptionreporter/nbproject/project.properties

```diff
@@ -15,3 +15,6 @@
 # specific language governing permissions and limitations
 # under the License.
 
+javac.source=1.8
+is.autoload=true
+is.enabled=false
```

Comment:
Does it make sense to specify `is.enabled` status for `is.autoload` module? Removing line with `is.enabled=...` shall have no effect, I assume.

Reply:
i actually completely misunderstood what autoload does :) If you are saying that setting autoload=true and not setting anything else will keep the modules off, i can give it another try tomorrow.

Reply:
The old explanation by @jglick is at:
- https://bits.netbeans.org/11.1/javadoc/org-openide-modules/org/openide/modules/doc-files/api.html
- just search for "autoload"

Surprisingly it is not included in the latest Javadoc:
- https://bits.netbeans.org/12.6/javadoc/org-openide-modules/org/openide/modules/doc-files/api.html
- I guess our Javadoc system broke somehow...

> setting autoload=true and not setting anything else will keep the modules off

Check `nbbuild/netbeans` and `platform/config/Modules/*.xml` with/without `is.enabled` setting. Possibly the `*xml` files stay unchanged. Then they would (obviously) behave the same in the runtime.

Reply:
https://bits.netbeans.org/11.1/javadoc/org-openide-modules/org/openide/modules/doc-files/api.html#enablement specifically

Reply:
@jglick do friend dependencies cause a module to load? trying to disable some findbugs modules atm.

Reply:
Yes, IIRC (and it has been a couple decades!) the only thing special about a “friend” is which packages it can access via class loader. No relation to enablement logic.